streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -583,6 +586,8 @@ class StreamingContext private[streaming] (
               scheduler.start()
             }
             state = StreamingContextState.ACTIVE
+            startTime = System.currentTimeMillis()
```

Comment:
My collaborator discussed on this in [here](https://github.com/apache/spark/pull/16000/files/651dc679b865603be677ca9d30b975ce5c3c5df0#diff-65e4a5a7796881a38e4a3940227506c5) with @vanzin, and I agreed with @vanzin's suggestion.